feature-libs/organization/schematics/add-organization/index_spec.ts

```diff
@@ -284,4 +328,26 @@ describe('Spartacus Organization schematics: ng-add', () => {
       });
     });
   });
+
+  describe('when other Spartacus features are already installed', () => {
```

Comment:
Shouldn't this test be part of misc tests or it should be reversed that if storefinder is installed then org features are using the same featureModules?

Reply:
I see similar tests in misc, so I guess it  wasn't adjusted here?